rest/blip_client_test.go

```diff
@@ -102,7 +119,14 @@ func (btr *BlipTesterReplicator) initHandlers(btc *BlipTesterClient) {
 			panic("no digest sent with proveAttachment")
 		}
 
-		attData, err := btc.getAttachment(digest)
+		collection := msg.Properties[db.BlipCollection]
+
+		// FIXME: Remove when we have CBG-2264 implemented
+		collection = btc.Collections[0]
+
+		btcr := btc.CollectionClients[collection]
```

Comment:
Can we reuse code from https://cs.github.com/couchbase/sync_gateway/blob/465adbdb0d3ce7062f53dc4c1bcbe122f7cde4a4/db/blip_handler.go?q=blipcollection#L132

or turn this into a function since it is duplicated 5x and we know we have to fix?

Reply:
Eh, maybe worth turning into a function but it'll be removed in a ticket this sprint and the FIXME's make it obvious